DOC.md

```diff
@@ -1812,7 +1812,9 @@ treesitter_postfix({
     },
 },{
     f(function(_, parent)
-        return ("std::move(%s)"):format(parent.snippet.env.LS_TSMATCH[1])
+        local node_content = table.concat(parent.snippet.env.LS_TSMATCH, '\n')
+        local replaced_content = ("std::move(%s)"):gsub("%%s", node_content)
```

Comment:
Ah, yeah that's better :+1:
:gsub -> :format?

Reply:
Yeah. In this case, `gsub` can be replaced by `format`.
Actually, these code I copied from another snippet `.fwd`. In that case, the template string is `std::forward<decltype(%s)>(%s)`.

Reply:
Ahh, I understand. Nice trick, shame format doesn't support named placeholders :D